aspnetcore/grpc/performance.md

```diff
@@ -201,7 +201,13 @@ message PayloadResponse {
 }  
 ```
 
-`ByteString` instances are created using `ByteString.CopyFrom(byte[] data)`. This method allocates a new `ByteString` and a new `byte[]`. Data is copied into the new byte array.
+Protobuf is a binary format so it can efficiently serialize large binary payloads without overhead. Compare this with text based formats like JSON which require [encoding bytes to base64](https://en.wikipedia.org/wiki/Base64) and adds 33% to the message size.
+
+When working with large `ByteString` payloads there are some best practices to avoid unnecessarily copies and allocations that are discussed below.
```

Comment:
```suggestion
When working with large `ByteString` payloads there are some best practices to avoid unnecessary copies and allocations that are discussed below.
```